scripts/css-processing.js

```diff
@@ -21,6 +21,12 @@ const postCSSPlugins = (resourcePath) => {
         require('postcss-inherit')(),
         require('postcss-preset-env')({
             stage: 0,
+            browsers: [
+                'last 2 Chrome versions',
+                'last 2 Firefox versions',
+                'last 2 Safari versions',
+                'last 2 iOS versions',
```

Comment:
Can you go with the last 3 iOS & Safari versions? While Chrome/Firefox auto-update, Safari can get blocked from updating on older OS versions, so it's a little more likely to be behind.

That doesn't mean we need to actively support those browsers, but I'd like to keep css from breaking because of this rule.

Reply:
I copied this from here: https://github.com/adobe/spectrum-css/blob/a3db903a3f6a1cab4b1c164d66f7eaf9813afe99/tools/component-builder/css/processors.js#L320-L328 but am by no means married to it.

I should also double check, but 2 versions of Chrome is roughly 12 - 18 weeks, where as 2 versions of Safari is roughly 2 years...

Will change, unless some of this changed your oppinion.

Reply:
Yeah -- it's roughly 2 years (but can be compounded by delayed OS updates), so the updates also lag quite a bit more for that last 5-10%.

Reply:
Cool. I'll bump that up. As long as it's "the same" enough, work for me!